daemons/fenced/fenced_commands.c

```diff
@@ -1390,18 +1390,19 @@ device_params_diff(GHashTable *first, GHashTable *second) {
 
     g_hash_table_iter_init(&gIter, first);
     while (g_hash_table_iter_next(&gIter, (void **)&key, (void **)&value)) {
+        char *other_value = NULL;
 
-        if(strstr(key, "CRM_meta") == key) {
+        if ((strstr(key, "CRM_meta") == key)
```

Comment:
`if (g_str_has_prefix(key, "CRM_meta") ...`, if you're already changing the `strcmp` call